docker-compose.yaml

```diff
@@ -161,3 +161,25 @@ services:
     logging:
       options:
         max-size: "100m"
+
+  rocksdb-backup:
+    container_name: rocksdb-backup
+    restart: always
```

Comment:
It's a one off, why do we need to restart always?

Reply:
it works based on an interval that we specify using env. in this case, it's true that it will immediately retrigger after restart, but shouldn't this be the intended behavior?

Reply:
Got it, it's working continuously atm. Wdyt of making it a one off - run it when you need it. When done - exit(0). As for me that's way easier to integrate on any ops side. If we run it continuously - we have to track when the backup is complete. If we stop it when it started the backup - we'll end up without a valid backup and so on.